optimum/exporters/openvino/model_patcher.py

```diff
@@ -65,53 +71,187 @@
 logger = logging.getLogger(__name__)
 
 
-for idx, spec in enumerate(UNSUPPORTED_OPS_PATCHING_SPEC):
-    if spec.name in {"repeat_interleave", "scaled_dot_product_attention"}:
-        UNSUPPORTED_OPS_PATCHING_SPEC.pop(idx)
+UNSUPPORTED_OPS_PATCHING_SPEC = [
+    PatchingSpec(torch, "tril", onnx_compatible_tril, torch.tril),
+    PatchingSpec(torch, "triu", onnx_compatible_triu, torch.triu),
+    PatchingSpec(torch, "rms_norm", onnx_compatible_rms_norm, torch.rms_norm),
+    PatchingSpec(torch.Tensor, "unfold", onnx_compatible_unfold, torch.Tensor.unfold),
+    PatchingSpec(torch.linalg, "norm", onnx_compatible_linalg_norm, torch.linalg.norm),
+    # TracerWarning: Using len to get tensor shape might cause the trace to be incorrect. Recommended usage would be tensor.shape[0]. Passing a tensor of different shape might lead to errors or silently give incorrect results.
+    PatchingSpec(torch.Tensor, "__len__", lambda x: x.shape[0], torch.Tensor.__len__),
+]
 
 
-BETTERTRANSFORMER_IGNORE = ["codegen", "gpt_neo"]
+class OVModelPatcher:
```

Comment:
what's the difference with `ModelPatcher` https://github.com/huggingface/optimum-onnx/blob/671b84f78a244594dd21cb1a8a1f7abb8961ea60/optimum/exporters/onnx/model_patcher.py#L455